cli/tests/constants.ts

```diff
@@ -2,14 +2,22 @@ import { Keypair } from "maci-domainobjs";
 
 import { homedir } from "os";
 
+// config params
 export const STATE_TREE_DEPTH = 10;
 export const INT_STATE_TREE_DEPTH = 1;
 export const MSG_TREE_DEPTH = 2;
 export const VOTE_OPTION_TREE_DEPTH = 2;
 export const MSG_BATCH_DEPTH = 1;
+export const pollDuration = 90;
+export const maxMessages = 25;
+export const maxVoteOptions = 25;
```

Comment:
Since the upper config params are in this kind of format ➡️ `MSG_BATCH_DEPTH`, are we going to unify them? (such as `POLL_DURATION`)

Reply:
can do yea

Reply:
feel like lowercase is easier to type though 😂